.gitignore

```diff
@@ -103,4 +103,10 @@ venv.bak/
 # mypy
 .mypy_cache/
 
-.idea/
\ No newline at end of file
+# local test files
+test_image_utils/
+.idea/
+
+# other
+maxfw/.DS_Store
```

Comment:
Could this be just `.DS_Store`? Because `.DS_Store` in all folders are likely to be ignored.